PC02_Graffiti_Griffith.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+'''
+Turtle starter code
+ATLS 1300
+Author: Dr. Z
+Author: YOUR NAME
+May 29, 2020
```

Comment:
Adding a description is always good practice!
```suggestion
May 29, 2020
This code draws a series of shapes on Jeff Bezos' face.
```